src/mono/Directory.Build.props

```diff
@@ -27,7 +27,7 @@
     <PackageVersion Condition="'$(PackageVersion)' == ''">5.0.0</PackageVersion>
 
     <!-- Set the boolean below to true to generate packages with stabilized versions -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
```

Comment:
@Anipik Can you please open an issue two remove these duplications?

Reply:
We need the the one in coreclr to ship non-stabilized versions of the packages with every servicing release. i can create an issue to remove the mono one.